modules/booksinput.js

```diff
@@ -0,0 +1,55 @@
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
```

Comment:
- [ ] I recommend adding the AddBook function as a method of the Book class otherwise you should remove this class because it does nothing. After adding the AddBook method you could export the class with `export default`. If you need help you could check this [article](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes).